cmds/tcz/tcz.go

```diff
@@ -116,10 +115,10 @@ func clonetree(tree string) error {
 
 func fetch(p string) error {
 
-	fullpath := path.Join(tczLocalPackageDir, p)
-	packageName := path.Join(tczServerDir, p)
+	fullpath := filepath.Join(tczLocalPackageDir, p)
+	packageName := filepath.Join(tczServerDir, p)
 	if _, err := os.Stat(fullpath); err != nil {
-		// path.Join doesn't quite work here. It will try to do file-system-like
+		// filepath.Join doesn't quite work here. It will try to do file-system-like
 		// joins and it ends up remove the // and replacing it with a clash.
 		cmd := "http://" + *host + ":" + *port + "/" + packageName
```

Comment:
side note (I know you didn't write this, but let's clean this up anyway): Could you change this to be a fmt.Sprintf? That's usually preferred in go.